test/util/buffered_readable_spec.js

```diff
@@ -50,7 +50,7 @@ describe('BufferedReadable', function() {
       assert(stream.push.calledTwice);
       assert.equal(stream.push.firstCall.args[0], 'abc');
       assert.equal(stream.push.secondCall.args[0], 'def');
-      assert.deepEqual(stream._buffer, ['def']);
+      assert.deepEqual(stream._buffer, []);
```

Comment:
Not sure if i understand this correctly, but it looks like after the 2nd call, 'def' should not be in the buffer. Otherwise the next call should give us back 'def'. Similarly, the first call gives us 'abc' since it is first in the original buffer.